api/PaymentMethodChangeEvent.json

```diff
@@ -50,6 +50,58 @@
           "deprecated": false
         }
       },
+      "PaymentMethodChangeEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/PaymentMethodChangeEvent/PaymentMethodChangeEvent",
+          "spec_url": "https://w3c.github.io/payment-request/#dom-paymentmethodchangeevent-constructor",
+          "description": "<code>PaymentMethodChangeEvent()</code> constructor",
+          "support": {
+            "chrome": {
+              "version_added": "76"
+            },
+            "chrome_android": {
+              "version_added": "76"
+            },
+            "edge": {
+              "version_added": "79"
+            },
+            "firefox": {
+              "version_added": false,
+              "notes": "Available only in nightly builds. Requires <code>dom.payments.request.enabled</code> to be set to <code>true</code> and the comma-delineated list in <code>dom.payments.request.supportedRegions</code> to contain one or more of the supported 2-character ISO locales, currently US and CA."
```

Comment:
Repeating the notes from the parent feature doesn't seem that helpful, and isn't done for the other entries.

Reply:
Fixed in next commit.